azure-pipelines.yml

```diff
@@ -14,7 +14,7 @@ resources:
     - repository: templates
       type: github
       name: shotgunsoftware/tk-ci-tools
-      ref: refs/heads/master
+      ref: refs/heads/release_to_appstore
```

Comment:
This will be removed before merging. Francis and I have been QAing using the new push to app store scripts that run on Azure.